src/vue/vuex-modules/data/auth-module.js

```diff
@@ -118,6 +120,9 @@ AuthModule.api = (moduleInstance, store) => {
           reject(new Error('Authentication is not enabled'))
         }
       })
+    },
+    getLoginLink: () => {
+
```

Comment:
Maybe we should print some warning message here for notification if someone will call it by mistake?

Reply:
good idea. will fix.

Reply:
actually this isn't needed at all ... forgot to remove it!